UI/Reports/balance_sheet.html

```diff
@@ -49,7 +49,7 @@ <h1><?lsmb text('Balance Sheet') ?></h1>
       <th colspan="<?lsmb max_path_depth ?>"
           style="font-weight:bold"> </th>
       <?lsmb FOREACH col IN report.sorted_col_ids -?>
-      <th><?lsmb report.cheads.ids.$col.props.description ?></th>
+      <th style="white-space:nowrap"><?lsmb report.cheads.ids.$col.props.description ?></th>
```

Comment:
Shouldn't we do this with css?

Reply:
Yes but we don't want it for all titles.

Reply:
Ok. So we need an extra css class?